generated/nidaqmx/_library_interpreter.py

```diff
@@ -2232,11 +2232,52 @@ def get_ai_chan_cal_exp_date(self, task, channel_name):
 
     def get_analog_power_up_states(
             self, device_name, channel_name, channel_type):
-        raise NotImplementedError
+        state = []
+
+        args = [device_name]
+        argtypes = [ctypes_byte_str]
+
+        for index in range(len(channel_name)):
```

Comment:
This isn't right. I gave some specific details in #298 . I think y'all know this, though, so I won't comment on all these issues.

Reply:
I have provided the information on the approach taken in #298 - https://github.com/ni/nidaqmx-python/pull/298#discussion_r1176130587

Reply:
OK, that makes sense.